src/test/sigopcount_tests.cpp

```diff
@@ -64,4 +65,16 @@ BOOST_AUTO_TEST_CASE(GetSigOpCount)
     BOOST_CHECK_EQUAL(p2sh.GetSigOpCount(scriptSig2), 3U);
 }
 
+
+BOOST_AUTO_TEST_CASE(blockSigOpAcceptLimit)
+{
+    BOOST_CHECK_EQUAL(Policy::blockSigOpAcceptLimit(0), 20000);
+    BOOST_CHECK_EQUAL(Policy::blockSigOpAcceptLimit(100), 20000);
+    BOOST_CHECK_EQUAL(Policy::blockSigOpAcceptLimit(50000), 20000);
+    BOOST_CHECK_EQUAL(Policy::blockSigOpAcceptLimit(1e6), 20000);
+    BOOST_CHECK_EQUAL(Policy::blockSigOpAcceptLimit(1e6 + 1), 40000);
+    BOOST_CHECK_EQUAL(Policy::blockSigOpAcceptLimit(2e6), 40000);
+    BOOST_CHECK_EQUAL(Policy::blockSigOpAcceptLimit(4e6 + 1), 100000);
```

Comment:
So when I see multiple checks like this, I think of table driven tests, they are far more clear, and easier to add additional cases. I generally do this with Golang not C++, not sure if that is something to strive for in the codebase. Here is an example in Golang: https://github.com/golang/go/wiki/TableDrivenTests